src/javascripts/ng-admin/Crud/list/ListController.js

```diff
@@ -47,9 +47,16 @@ define(function () {
 
         this.ReadQueries
             .getAll(this.view, page, true, this.search, this.sortField, this.sortDir)
-            .then(function (nextData) {
```

Comment:
You also need to fix parameters passed to `getAll`method. The third one (`true`) does not exists anymore.